### ISSUE TITLE
Keyboard font: Proposing reimplementation for WinKey symbol 

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [alerque/libertinus#449]
**Describe the bug**
Seem from #416, I think the Libertinus Keyboard should update the design of WinKey to adopt the 2012 Windows logo design, then the old glyph is preserve as `hist` feature.

**Expected vs. actual behavior**
By default, the WinKey symbol should including 2012 Windows logo, while `hist` feature enabled, the 2001 logo design is present.

**Screenshots / logs**
— Non (plus ultra)

**Additional context**

**Comment by KrasnayaPloshchad** [comment]
I think you should even update to follow Windows 11 logo.

**Comment by fitojb** [comment]
Well, the Win11 logo is four rectangles, which is of course both uncopyrightable and timeless; this project could settle on that.

**Comment by nsemrau** [comment]
Yes, I thought so as well. Incidentally, this is also the way Wikipedia [does it](https://en.wikipedia.org/wiki/Windows_key#Licensing): »Wikipedia uses the [Unicode](https://en.wikipedia.org/wiki/Unicode) character [U+](https://en.wikipedia.org/wiki/Mathematical_Operators)229E ⊞ [SQUARED PLUS](https://en.wikipedia.org/wiki/Mathematical_operators_and_symbols_in_Unicode) as a simulation of the logo.« Seeing the newest flat style iteration inspired by Windows 11, I would find this quite fitting.
